### PR TITLE
Completed html-block-and-inline. Bryan Valdovinos

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
Screen Shot 2022-05-09 at 3 21 33 PM

### CODE REVIEW

File: html-block-and-inline/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>
+    <div>Price: <s>$19.99</s> $14.99</div><br/>
+    <div>Part of our new Americana collection, this <em>beatuiful</em> blue is alrady a customer favorite!</div><br/>
+    <div>Enjoy complete <strong>Freedom of Movement!</strong></div><br/>
+    <div>Does not slip and slide like a regular blanket and keeps you warm from head to toe.</div><br/>
```
Comment (codingcodymiller) [inline html-block-and-inline/index.html:13]: `div` elements should not be used as text elements. If you want normal-sized default text, you should just use a `p` element. Also, the `br` elements are unnecessary, you should remove them.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by bryanvaldovinos** [comment]
Screen Shot 2022-05-09 at 3 38 05 PM

File: html-block-and-inline/index.html (hunk 1 above)
Comment (bryanvaldovinos) [reply html-block-and-inline/index.html]: Just fixed, thanks!

File: html-block-and-inline/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <div>
+      <p>Price: <s>$19.99</s> $14.99</div>
```
Comment (codingcodymiller) [inline html-block-and-inline/index.html:10]: Whoops, this has a closing `div` tag, but it should be a closing `p` tag

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-block-and-inline/index.html (hunk 2 above)
Comment (bryanvaldovinos) [reply html-block-and-inline/index.html]: removed, thank you.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**